server/sonar-server/src/main/java/org/sonar/server/test/index/TestIndex.java

```diff
@@ -71,7 +72,7 @@ public TestIndex(EsClient client) {
       .setTypes(TestIndexDefinition.TYPE)
       .setSize(searchOptions.getLimit())
       .setFrom(searchOptions.getOffset())
-      .setQuery(QueryBuilders.filteredQuery(QueryBuilders.matchAllQuery(), FilterBuilders.termFilter(FIELD_FILE_UUID, testFileUuid)));
+      .setQuery(QueryBuilders.filteredQuery(QueryBuilders.matchAllQuery(), termQuery(FIELD_FILE_UUID, testFileUuid)));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "filteredQuery"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)